tsconfig.json

```diff
@@ -22,7 +22,13 @@
     "lib": [
       "ES2022",
       "dom"
-    ]
+    ],
+    "strictPropertyInitialization": false,
+    "paths": {
+      "@src/*": [
+        "src/*"
+      ]
+    }
```

Comment:
This is useless. `paths` property can be used to make shortcuts, but it's not the case with `@src: src`